Buildings/Fluid/ZoneEquipment/Data/FCUSizing.mo

```diff
@@ -0,0 +1,84 @@
+within Buildings.Fluid.ZoneEquipment.Data;
```

Comment:
Maybe we should create a sub-package called FanCoil and put the Data package under it